openmetadata-ui/src/main/resources/ui/src/pages/MyDataPage/MyDataPage.component.tsx

```diff
@@ -342,14 +384,18 @@ const MyDataPage = () => {
       !isUndefined(countTables) &&
       !isUndefined(countTopics) &&
       !isUndefined(countDashboards) &&
-      !isUndefined(countPipelines) ? (
+      !isUndefined(countPipelines) &&
```

Comment:
isUndefined is checks only undefined 
please refer https://lodash.com/docs/4.17.15#isUndefined

Reply:
Yes and here we only need to check explicitly for undefined, because when fetching count we already handle falsy values with 0 before setting the count states